plugins/node/opentelemetry-plugin-mysql/src/utils.ts

```diff
@@ -16,7 +16,8 @@
 
 import { Attributes } from '@opentelemetry/api';
 import { AttributeNames } from './enums';
-import { ConnectionConfig, Query } from './types';
+import { Query } from './types';
+import { ConnectionConfig, PoolActualConfig } from 'mysql';
```

Comment:
```suggestion
import type { ConnectionConfig, PoolActualConfig } from 'mysql';
```

Reply:
Thanks, what's the difference?

Reply:
ah TIL https://www.typescriptlang.org/docs/handbook/modules.html#importing-types